CMakeLists.txt

```diff
@@ -49,6 +49,14 @@ if(CROSS_COMPILE)
 	# Raspbian ships CLang 3.9 in its repositories
 	set(CLANG_FOUND /usr/bin/clang-3.9)
 	set(ADVANCED_OPENCL_FEATURES ON)
+        if (SYSROOT_CROSS)
+          # set(CMAKE_SYSROOT ${SYSROOT_CROSS})
+          set(CMAKE_FIND_ROOT_PATH ${SYSROOT_CROSS})
+          set(SYSROOT_LIBRARY_FLAGS "-Wl,-rpath-link,${SYSROOT_CROSS}/lib/arm-linux-gnueabihf:${SYSROOT_CROSS}/usr/lib/arm-linux-gnueabihf")
```

Comment:
why do you add the sysroot-cross paths to the runtime search directories?
The target machine does not have these paths.

Reply:
To keep the cross-compiler directries clean.
Original crosstool-ng avoid to add external libraries to the compiler directories directly.

To keep the rule, we need to add external path.